dpdata/plugins/lammps.py

```
@@ -119,11 +121,16 @@ def to_system(self, data, file_name: FileType, frame_idx=0, **kwargs):
             The output file name
         frame_idx : int
             The index of the frame to dump
+        output_spins : bool, optional
+            Whether to output spin information columns. Default is False.
+            When True, outputs additional columns for spin direction and magnitude.
         **kwargs : dict
             other parameters
         """
         assert frame_idx < len(data["coords"])
```

Ruff (0.12.2)

Use of `assert` detected

(S101)